react/features/mobile/external-api/middleware.js

```diff
@@ -2,6 +2,8 @@
 
 import { NativeEventEmitter, NativeModules } from 'react-native';
 
+import { ENDPOINT_TEXT_MESSAGE_NAME } from '../../../../modules/API/constants';
+import logger from '../../analytics/logger';
```

Comment:
Don't use a random logger, create a new one for this feature if you need to please.